Formula/telnet.rb

```diff
@@ -0,0 +1,46 @@
+class Telnet < Formula
+  desc "Apple telnet built from macOS 10.12.4 sources"
```

Comment:
Maybe just `User interface to the TELNET protocol` as it's described in the man page